src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -964,7 +968,57 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
     }
   }
 
-  onUseSavedCardChange(event: MatCheckboxChange, paymentMethod: PaymentMethod) {
+  progressFromStepOne() {
+
+    const control = this.donationForm.controls['amounts'];
+    if(! control!.valid) {
+        this.snackBar.open(
+          this.displayableAmountsStepErrors() || 'Sorry, there was an error with the donation amount',
+            undefined,
+            { duration: 5_000, panelClass: 'snack-bar' }
+        );
+
+      return;
+    }
+
+    this.next();
+  }
+
+    public displayableAmountsStepErrors = () => {
+        const errors = this.donationAmountField?.errors;
+
+        if (!errors) {
+            return '';
```

Comment:
not sure if we really want to return '' here - think we may have put this in as a shortcut to getting the type check to pass last week. Maybe throw an error instead? Maybe that's just changing the ? to ! on 988.

Reply:
ah see it's documented as null if there are no errors, so actually this seems fine.